### PR TITLE
 Issue #11589: adds descriptions for ResultOfMethodCallIgnored

### PR DESCRIPTION
**Pull request by rnveach**
#11589

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java [hunk 1]
```diff
@@ -335,7 +335,7 @@ public void testChangeInConfig() throws Exception {
      * until #11589
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java:335]: should be removed.

File: src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java [hunk 2]
```diff
@@ -335,7 +335,7 @@ public void testChangeInConfig() throws Exception {
...
      *
      * @noinspection ResultOfMethodCallIgnored
-     * @noinspectionreason ResultOfMethodCallIgnored - temporary suppression until #11589
+     * @noinspectionreason ResultOfMethodCallIgnored - Setup for mockito for which method to mock.
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java:338]: can we be specific , something like `toByteArray only mocked to throw exception`

**Review by romani — CHANGES_REQUESTED** [review]
items:

File: src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java (hunk 1 above)
Comment (rnveach) [reply src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java:335]: Done.

**Review by romani — APPROVED** [review]

**Pull request merged**